apps/web/src/stories/Button.tsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import './button.css';
+
+interface ButtonProps {
+  /**
+   * Is this the principal call to action on the page?
+   */
+  primary?: boolean;
+  /**
+   * What background color to use
+   */
+  backgroundColor?: string;
+  /**
+   * How large should the button be?
+   */
+  size?: 'small' | 'medium' | 'large';
+  /**
+   * Button contents
+   */
+  label: string;
+  /**
+   * Optional click handler
+   */
+  onClick?: () => void;
+}
+
+/**
+ * Primary UI component for user interaction
+ */
```

Comment:
Take note of the syntax for all of these comments. It uses this special character set to prefix each multiline comment:

```
/**
```

This is called JSDocs and it is a special kind of comment that can be referenced from the calling scope of the code and use to automate documentation. In our codebase all component props should be commented with JSDocs, just like this example. 

Additionally, all functions should be documented like this:

```ts
/**
* A simple function that adds its two parameters together
*
* @param { number } num1 - the first number to add
* @param { number } num2 - the second numebr to add
* @returns { number }
*/
const sum = (num1: number, num2: number) : number => num1 + num2;
```
Functions should document all of their params, including name and type as well as their return type. Additionally they should have a brief description of what the function does.

You can add the [JSDoc Extension](https://marketplace.visualstudio.com/items?itemName=lllllllqw.jsdoc) to VSCode and easily generate JSDocs templates:

![js_docs](https://user-images.githubusercontent.com/12022922/219900362-81289f4d-95e1-4935-a28f-801d68b51569.gif)